### ISSUE TITLE
Loss curves for VQGAN looking not too good

### ISSUE THREAD

**Issue by NonsansWD** [Alescontrela/viper_rl#9]
![image](https://github.com/user-attachments/assets/a753813f-141d-44fb-bf02-e7c38cbcc660)

As you can see in the image I provided, the losses are really sky rocketing to up to 10^7 so I wanted to ask if this is fine or is this some kind of overfitting or something and should I be worried about these curves? The reconstructed images have been all over the place from completely yellow images to completely green to completely pink and sometimes with some white or other colored spots so nowhere close to the actual image in the batch. Can you explain what might happen here?

**Comment by yangzhao-666** [comment]
Hi, did you manage to train the videoGPT? May I ask how much compute / time is needed roughly to train it?
